detection_rules/schemas/__init__.py

```diff
@@ -250,9 +256,9 @@ def get_stack_schemas(stack_version: Optional[str] = '0.0.0') -> OrderedDictType
     return versions_reversed
 
 
-def get_stack_versions(drop_patch=True, reverse=True) -> List[str]:
+def get_stack_versions(drop_patch=False) -> List[str]:
     """Get a list of stack versions supported (for the matrix)."""
-    versions = reversed(get_stack_schemas()) if reverse else list(get_stack_schemas())
+    versions = list(load_stack_schema_map())
```

Comment:
loading the file directly as opposed to loading the versions from `get_stack_schemas` is what resolved the bug, since that function limits to the "current" branch